### ISSUE TITLE
Returning to previous Activity when orientation changes

### ISSUE THREAD

**Issue by dininski** [aksonov/react-native-router-flux#1308]
### Version
Tell us which versions you are using: 

- react-native-router-flux v3.36.0
- react-native v0.34.1

### Expected behaviour

When in an action that is not a part of tabs screen and changing the rotation the previous scene is rendered

### Actual behaviour

When changing the rotation the same scene should be rendered.

### Steps to reproduce

My router config looks like this:

            <Provider store={store}>
                <RouterWithRedux getSceneStyle={getSceneStyle}>
                        <Scene hideNavBar key="root">
                            <Scene key="Login" title="Login" component={Scenes.Login} initial/>
                            <Scene key="Register" title="Register" component={Scenes.Register}/>
                            {getScene({key: 'AudiobookListen', component: Scenes.AudiobookListen, title: 'test', hideNavBar: true})}
                            <Scene key="Main" initial>
                                <Scene key="drawer" component={Drawer}>
                                    <Scene key="drawerMain" tabs>
                                        {getScene({key: 'AudiobooksMain', component: Scenes.AudiobooksMain, title: 'All books'})}
                                        {getScene({key: 'Settings', component: Scenes.Settings, title: 'Settings'})}
                                        {getScene({key: 'Search', component: Scenes.Search, title: 'Search'})}
                                        {getScene({key: 'MyBooks', component: Scenes.MyBooks, title: 'My books'})}
                                        {getScene({key: 'Browse', component: Scenes.Browse, title: 'Browse'})}
                                    </Scene>
                                </Scene>
                        </Scene>
                    </Scene>
                </RouterWithRedux>
            </Provider>

where `getScene` simply adds an additional prop to the Scene and returns a component generated using `React.createComponent`.

In the AudiobooksMain action I have some logic which on clicking an item fires the `Actions.AudiobookListen` action. Once there if you rotate the screen you are immediately returned to the `AudiobooksMain` screen.

**Comment by dininski** [comment]
It turns out that it was an issue with the drawer incorrectly calling onClose and refreshing the Main view

**Comment by gsavvid** [comment]
@dininski I'm having the same problem and I'm suspecting it's also related to the drawer. However, I still can't find the solution. Could you please share what solved it for you?

**Comment by dininski** [comment]
@gsavvid of course
In my case when rotating the device for some reason the Drawer was still "bound" to the previous view and was executing the default `onOpen` and `onClose` callbacks with the previous Action as a context  trying to refresh the previous view. I used the code from the default sample in the docs:

```
                onOpen={()=>Actions.refresh({key:state.key, open: true})}
                onClose={()=>Actions.refresh({key:state.key, open: false})}
```

however removing those callbacks where I set up my drawer component fixed the issue.
I was using this - https://github.com/aksonov/react-native-router-flux/blob/master/docs/OTHER_INFO.md#drawer-side-menu-integration but with the `onOpen` and `onClose` handlers removed.
